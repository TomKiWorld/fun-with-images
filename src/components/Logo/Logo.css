.Logo {
  height: 6rem;
  width: 6rem;
  position: absolute;
  top: 0;
  left: 1rem;
}

.Logo img {
  width: 100%;
}
