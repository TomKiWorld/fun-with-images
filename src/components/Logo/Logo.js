import React from 'react';
import './Logo.css';
import funImages from './funGif.gif';

/**
 * Site logo
 */
const Logo = () => {
  return (
    <div className='Logo'>
      <img src={funImages} alt='logo'/>
    </div>
  );
}

export default Logo;
