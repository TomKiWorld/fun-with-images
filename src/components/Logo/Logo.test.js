import { shallow } from 'enzyme';
import React from 'react';
import Logo from './Logo';

describe('Logo Component', () => {
  it('Expect to render Logo Component', () => {
    expect(shallow(<Logo />)).toMatchSnapshot();
  });
});
