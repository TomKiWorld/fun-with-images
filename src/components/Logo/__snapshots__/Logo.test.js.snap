// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component Expect to render Logo Component 1`] = `ShallowWrapper {}`;
